applications/GeoMechanicsApplication/custom_utilities/matrix_equation_term_utilities.cpp

```diff
@@ -0,0 +1,17 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+#include "matrix_equation_term_utilities.h"
+
+namespace Kratos::Geo::MatrixEquationTermUtilities
```

Comment:
Richard, a few days ago Wijtze-Pieter gave two options to replace the drainage policy with transport and water. I think TransportEquationUtilities is more descriptive than MatrixEquation...

Reply:
Good point, indeed that is a more suitable name, provided we only put the calculation of the C, H and Q matrices here and leave M, D and K to another namespace (since they are related to the equation of motion and are not really related to transport equations).